Firestore/Example/Tests/API/FIRFirestoreTests.mm

```diff
@@ -63,4 +63,16 @@ - (void)testDeleteApp {
                                }];
 }
 
+- (void)testSetEmulatorSettingsSetsHost {
+  // Ensure the app is set appropriately.
+  FIRApp *app = testutil::AppForUnitTesting();
+
+  FIRFirestore *firestore = [FIRFirestore firestoreForApp:app];
+
+  [firestore useEmulatorWithHost:@"localhost" port:1000];
+
+  NSString *host = [firestore valueForKeyPath:@"settings.host"];
```

Comment:
Given that these are all properties, this seems like it could be more more concisely spelled as `firestore.settings.host`. This has the benefit of making this show up when when searching for usages.

Reply:
Done.